### PR TITLE
make relocatable

### PR DESCRIPTION
**Pull request by jbrea**

### CODE REVIEW

**Comment by t-bltg** [comment]
Dropping a PR without any kind of description explaining at least why it is needed is not really the way to contribute.
For the CI to pass you'd need to rebase your PR against current master or add a merge commit. Thanks.

**Comment by jbrea** [comment]
Oh, sorry, I thought it was self-evident :) (I did the same for a few other packages).
The problem is that `@__DIR__` doesn't play well with the `PackageCompiler` without `RelocatableFolders` (see [here](https://github.com/JuliaPackaging/RelocatableFolders.jl)).

**Review by t-bltg — APPROVED** [review]
LGTM (pending CI - TIL `RelocatableFolders`).

**Comment by jbrea** [comment]
Thanks @t-bltg!

File: Project.toml [hunk 1]
```diff
@@ -57,6 +58,7 @@ PyPlot = "2"
 RecipesBase = "^1.2"
 RecipesPipeline = "0.5 - 0.6"
 Reexport = "0.2, 1"
+RelocatableFolders = "0.3"
```
Comment (t-bltg) [inline Project.toml:61]: @jbrea, is there a reason for not supporting `RelocatableFolders` 1.X ?

Comment (t-bltg) [reply Project.toml:61]: Fixed by https://github.com/JuliaPlots/Plots.jl/pull/4338.

**Pull request merged**